integration/simple-cache-client.test.ts

```diff
@@ -31,6 +33,9 @@ const cacheClientProps: SimpleCacheClientProps = {
 const INTEGRATION_TEST_CACHE_NAME =
   process.env.TEST_CACHE_NAME || 'js-integration-test-default';
 
+const LOL_BYTE_ARRAY = Uint8Array.of(108, 111, 108);
+const FOO_BYTE_ARRAY = Uint8Array.of(102, 111, 111);
```

Comment:
Put these inside the describe block where they're used?

Reply:
sure.